### ISSUE TITLE
configmaps aws_auth connection refused

### ISSUE THREAD

**Issue by tnguyen14** [hashicorp/learn-terraform-provision-eks-cluster#7]
I tried running `terraform apply`, and got this error.
```
Error: Post http://localhost/api/v1/namespaces/kube-system/configmaps: dial tcp 127.0.0.1:80: connect: connection refused

  on .terraform/modules/eks/terraform-aws-eks-11.1.0/aws_auth.tf line 62, in resource "kubernetes_config_map" "aws_auth":
  62: resource "kubernetes_config_map" "aws_auth" {
```

What could I be doing wrong here? I originally opened https://github.com/terraform-aws-modules/terraform-aws-eks/issues/853, but was informed that it could be an issue with this repo.

**Comment by tnguyen14** [comment]
Just realized this is a duplicate of #5

**Comment by im2nguyen** [comment]
This has been addressed in #2  and a change has been merged into master, thanks for flagging this
